### PR TITLE
Add readme

### PR DESCRIPTION
**Pull request by sbrunner**
Be able to generate a file without writing a configuration file
Add issue on the limitations

### CODE REVIEW

File: jsonschema_gentypes/cli.py [hunk 1]
```diff
@@ -28,14 +29,26 @@ def main() -> None:
...
-    try:
-        config = validate.load(args.config, json.loads(schema_data))
-    except validate.ValidationError as error:
-        LOG.error(error)
+    if args.python is not None or args.json_schema is not None:
+        if args.python is None or args.json_schema is None:
+            print("If you specify the argument --python or --json-schema the other one is required")
+            sys.exit(1)
+        config: configuration.Configuration = {
+            "generate": [{"source": args.json_schema, "destination": args.python}]
```
Comment (arnaud-morvan) [inline jsonschema_gentypes/cli.py:41]: As I understand, this use case is a sort of simple mode, we only give an input schema and a python output on the command line and config argument is completely ignored. No possibility to configure headers, callbacks, root_name, etc.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,66 @@
...
+
+You can also run it with Docker:
+
+```bash
+docker run --rm --user=$(id --user) --volume=$(pwd):/src camptocamp/jsonschema-gentypes
+```
+
+## Config file
+
+Cou can also vrite a config file named `jsonschema-gentypes.yaml` with:
```
Comment (arnaud-morvan) [inline README.md:29]:
```suggestion
You can also write a config file named `jsonschema-gentypes.yaml` with:
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,66 @@
...
+
+```yaml
+headers: >
+    # Automatically generated file from a JSON schema
+# Used to correctly format the generated file
+callbacks:
+    - - black
+    - - isort
+generate:
+    - # Json schema file name
```
Comment (arnaud-morvan) [inline README.md:39]:
```suggestion
    - # JSON schema file path
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,66 @@
...
+      source: jsonschema_gentypes/schema.json
+      # Python file name
```
Comment (arnaud-morvan) [inline README.md:41]:
```suggestion
      # Python file path
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,66 @@
...
+      # The name of the root element
+      root_name: Config
+      # Argument passed to the API
+      api_arguments:
+          additional_properties: Only explicit
+      # Rename an element
+      name_mapping: {}
+```
+
+## Limitation
```
Comment (arnaud-morvan) [inline README.md:52]:
```suggestion
## Limitations
```

File: README.md [hunk 6]
```diff
@@ -1,3 +1,66 @@
...
+
+Requires Python 3.8
+
+See [issues](https://github.com/camptocamp/jsonschema-gentypes/issues?q=is%3Aissue+is%3Aopen+label%3Alimitation) contains the limitations
```
Comment (arnaud-morvan) [inline README.md:56]:
```suggestion
See [issues](https://github.com/camptocamp/jsonschema-gentypes/issues?q=is%3Aissue+is%3Aopen+label%3Alimitation) with label "limitation"
```

File: jsonschema_gentypes/cli.py (hunk 1 above)
Comment (sbrunner) [reply jsonschema_gentypes/cli.py:41]: Yes, it's just tu have a quick comment that didn't need to create a new file...

File: README.md [hunk 7]
```diff
@@ -1,3 +1,66 @@
...
+      # Python file path
+      destination: jsonschema_gentypes/configuration.py
+      # The name of the root element
+      root_name: Config
+      # Argument passed to the API
+      api_arguments:
+          additional_properties: Only explicit
+      # Rename an element
+      name_mapping: {}
+```
```
Comment (arnaud-morvan) [inline README.md:50]: I would add an example of command with config file here.

**Review by arnaud-morvan — APPROVED** [review]
Minor comment about run with config file in README.

**Pull request merged**